internal/pkg/template/templates/environment/cf.yml

```diff
@@ -288,7 +288,7 @@ Resources:
 {{include "custom-resources-role" . | indent 2}}
   EnvironmentHostedZone:
     Metadata:
-      'aws:copilot:description': 'A Route 53 Hosted Zone for the subdomain ${EnvironmentName}.${AppName}.${AppDNSName}'
+      'aws:copilot:description': "A Route 53 Hosted Zone for the environment's subdomain"
```

Comment:
sounds good, it'd be amazing if we can have a follow-up PR that writes the domain name 🥺